### DISCUSSION TITLE
DBeaver Formatting Issue

### DISCUSSION THREAD

**Discussion by daniellee789** [dbeaver/dbeaver#22945] [Q&A]
Hello, I recently installed DBeaver on a new computer and have been having a formatting issue. DBeaver is automatically italicizing some of the qualifiers/aliases. I have looked through all of the settings and I have tried changing all of the fonts multiple times, but it is still doing this. I did not have this issue on my previous computer. Is there a setting I am missing?

![Screenshot 2024-02-23 114541](https://github.com/dbeaver/dbeaver/assets/83609818/c00e81d4-8879-4936-b0e2-b0ce1d9206a5)

**Comment by E1izabeth** [comment]
This is a feature to have visual differentiation for identifies coming from the database schema (like real table name or its column) and query-local identifiers (like names used as table alias, column projection alias and CTE WITH-clause query name). 
You can also ctrl-click any cursived identifier to jump to its definition in a query the same way how it works for ctrl-clicked table name jumping to the table definition from the schema.
We don't have any setting to turn italicizing off at the moment, just to change the color of the corresponding identifier class.

**Reply by cpcurtis** [reply]
My old version of DBeaver didn't do this. Now that I have (finally) upgraded the versionI am using, I really don't like the italics. I see where I can change the color, and have done so, but the italics is not consistent for me as I am writing code (particularly where it auto-italicizes CTE names) so I would like to turn it off. 

Commenting to hopefully indicate this is still a problem for some users.
